### ISSUE TITLE
DeviceList() is empty

### ISSUE THREAD

**Issue by andycui66** [truenas/py-SMART#39]
I can create a device with `d = Device('/dev/nvme0')` But DeviceList() returns empty list.

```
>>> from pySMART import DeviceList
>>> dl = DeviceList()
>>> dl.devices
[]
```

Python 3.8.5
pySMART == 1.0
smartctl 7.1

**Comment by ralequi** [comment]
Check this first:

1) Have you tried to run your code on root?
2) Run with the same user as your python the following command `smartctl --scan-open`. If it says you are not allowed to open the devices you probably have to run it on root

**Comment by MichaIng** [comment]
Might be related to and hence fixed with #41.

**Comment by ralequi** [comment]
Closing due to lack of activity.

Please, reopen this issue if you could give more detailed info.
